worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +54,39 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
+	std::string playerguess = "";
+	int likescore = 0;
```

Comment:
Do `likescore` and `playerguess` need to be declared outside the loop? Usually better to declare variables close to the point of first use.